tests/request_replayer/index.php

```diff
@@ -1,15 +1,21 @@
 <?php
 
-$requestLog = '/tmp/php_request_replayer_' . getmypid() . '.json';
+$defaultRequestLog = '/tmp/php_request_replayer_' . getmypid() . '.json';
```

Comment:
This will work on systems with temp dirs in other locations. :)
```suggestion
$requestLog = tempnam(sys_get_temp_dir(), 'php_request_replayer_');
```

Reply:
Actually, we don't want to create the file here if we don't need to (which `tempnam()` does) so this is better. :)
```suggestion
$requestLog = sys_get_temp_dir() . '/php_request_replayer_' . getmypid() . '.json';
```